apps/smartling/frontend/package.json

```diff
@@ -12,8 +12,6 @@
     "@types/webpack-env": "1.18.5",
     "cross-env": "^7.0.3",
     "fetch-mock": "^9.0.0-beta.2",
-    "jest-fetch-mock": "^3.0.1",
```

Comment:
removing some unused dependencies I caught on this round. I think a lot of these are duplicated bt the frontend and the lambda and not actually used on the frontend. contentful-management was showing up as still being a frontend vulnerability for this app but it's apparently not actually in use on the frontend.